pkg/volume/plugins.go

```diff
@@ -119,6 +119,39 @@ type VolumePluginMgr struct {
 	plugins map[string]VolumePlugin
 }
 
+// InternalVolume is an internal representation of a volume.  All API volume types translate to InternalVolume.
+type InternalVolume struct {
```

Comment:
yes, missed that on the 2nd commit, but refresh and take a look at the 3rd.

----- Original Message -----
From: "Tim Hockin" <notifications@github.com>
To: "GoogleCloudPlatform/kubernetes" <kubernetes@noreply.github.com>
Cc: "Mark Turansky" <mturansk@redhat.com>
Sent: Tuesday, April 14, 2015 2:07:22 PM
Subject: Re: [kubernetes] Refactor Volumes to use InternalVolume instead of API types (#6809)

> @@ -119,6 +119,39 @@ type VolumePluginMgr struct {
>  	plugins map[string]VolumePlugin
>  }
>  
> +// InternalVolume is an internal representation of a volume.  All API volume types translate to InternalVolume.
> +type InternalVolume struct {

Still suggest a rename.  `type Spec` or `type APISpec` or `type FromAPI` or something

---
Reply to this email directly or view it on GitHub:
https://github.com/GoogleCloudPlatform/kubernetes/pull/6809/files#r28353186